lib/capybara/poltergeist/browser.rb

```diff
@@ -260,18 +260,25 @@ def path(page_id, id)
       command 'path', page_id, id
     end
 
-    def network_traffic
-      command('network_traffic').values.map do |event|
-        NetworkTraffic::Request.new(
-          event['request'],
-          event['responseParts'].map { |response| NetworkTraffic::Response.new(response) },
-          event['error'] ? NetworkTraffic::Error.new(event['error']) : nil
-        )
+    def network_traffic(type = nil)
+      if type == :blocked
+        command('blocked_requests').values.map do |event|
+          NetworkTraffic::Request.new(event['request'])
+        end
+      else
+        command('network_traffic').values.map do |event|
+          NetworkTraffic::Request.new(
+            event['request'],
+            event['responseParts'].map { |response| NetworkTraffic::Response.new(response) },
+            event['error'] ? NetworkTraffic::Error.new(event['error']) : nil
+          )
+        end
       end
     end
 
     def clear_network_traffic
       command('clear_network_traffic')
+      command('clear_blocked_requests')
```

Comment:
I don't think we need separate commands for this -- just have the clear_network_traffic clear it all